mne/io/base.py

```diff
@@ -1565,7 +1565,8 @@ def plot_psd(self, tmin=0.0, tmax=None, fmin=0, fmax=np.inf,
                  proj=False, n_fft=2048, picks=None, ax=None,
                  color='black', area_mode='std', area_alpha=0.33,
                  n_overlap=0, dB=True, average=True, show=True,
-                 n_jobs=1, line_alpha=None, verbose=None):
+                 n_jobs=1, line_alpha=None, spatial_colors=False,
```

Comment:
Assuming the spatial colors are actually nice, I'm +1 for actually setting a deprecation cycle None-as-False->True for this. We're already deprecating `tmax` for the `raw.psd` so people will already be pinged about one deprecation when using the defaults.